test/build-turbopack-tests-manifest.js

```diff
@@ -66,11 +72,11 @@ async function updatePassingTests() {
   }
 
   for (const info of Object.values(passing)) {
-    info.failed = [...new Set(info.failed)]
-    info.pending = [...new Set(info.pending)]
+    info.failed = [...new Set(info.failed)].sort()
```

Comment:
Without sorting these multiple runs of the sync script end up reordering the `failed` `pending` and `passed`, even though there were effectively no changes.